vue/src/components/Board.vue

```diff
@@ -72,12 +76,12 @@ export default {
   components: { },
   props: ['notes'],
   methods: {
-    addNote(newNote) {
-      //Send up to parent
-      this.$emit('add-note', newNote);
+    // Returns important parts of integrated HTML code
+    renderText: function (text) {
+      return text.slice(1, text.length - 1)
```

Comment:
Substring works on :8080 but not on :1337. When testing it note content was saved without "". The substring was calculated and the text that was displayed had "p>" at its beginning because the first letter was "<".